### PR TITLE
Tips-Tricks - retrieving broker configuration when downscale running

### PR DESCRIPTION
**Pull request by bartam1**
https://github.com/banzaicloud/koperator/pull/824

### CODE REVIEW

**Review by pregnor — APPROVED** [review]

File: docs/tips-tricks.md [hunk 1]
```diff
@@ -22,3 +22,10 @@ When the **headlessServiceEnabled** option is enabled (true) in your KafkaCluste
 
 When the **headlessServiceEnabled** option is disabled (false), it creates a ClusterIP service. When using a ClusterIP service, your client application doesn’t need to be aware of every Kafka broker endpoint, it simply connects to *kafka-all-broker:29092* which covers dynamically all the available brokers. That way if the Kafka cluster is scaled dynamically, there is no need to reconfigure the client applications.
 
+## Retrieving broker configuration when downscale running
+
+When a broker is on downscaling thus the broker configuration is missing from the kafkaCluster/spec/brokers field, the broker last configuration can be retrieved with the following command.
```
Comment (Kuvesz) [inline docs/tips-tricks.md:27]:
```suggestion
When a broker is downscaling thus the broker configuration is missing from the kafkaCluster/spec/brokers field, the last broker configuration can be retrieved with the following command.
```

**Review by Kuvesz — CHANGES_REQUESTED** [review]

File: docs/tips-tricks.md [hunk 2]
```diff
@@ -22,3 +22,10 @@ When the **headlessServiceEnabled** option is enabled (true) in your KafkaCluste
 
 When the **headlessServiceEnabled** option is disabled (false), it creates a ClusterIP service. When using a ClusterIP service, your client application doesn’t need to be aware of every Kafka broker endpoint, it simply connects to *kafka-all-broker:29092* which covers dynamically all the available brokers. That way if the Kafka cluster is scaled dynamically, there is no need to reconfigure the client applications.
 
+## Retrieving broker configuration during downscale running
```
Comment (stoader) [inline docs/tips-tricks.md:25]:
```suggestion
## Retrieving broker configuration during downscale is running
```

**Review by stoader — CHANGES_REQUESTED** [review]

**Review by Kuvesz — APPROVED** [review]

File: docs/tips-tricks.md (hunk 2 above)
Comment (Kuvesz) [reply docs/tips-tricks.md:25]: This one sounds weird to me, your earlier suggestion would be best I think.

File: docs/tips-tricks.md [hunk 3]
```diff
@@ -22,3 +22,10 @@ When the **headlessServiceEnabled** option is enabled (true) in your KafkaCluste
...
+
+When a broker is downscaling thus the broker configuration is missing from the kafkaCluster/spec/brokers field, the last broker configuration can be retrieved with the following command.
```
Comment (fekete-robert) [inline docs/tips-tricks.md:27]:
```suggestion
When a broker is downscaling, the broker configuration is missing from the kafkaCluster/spec/brokers field. You can retrieve the last broker configuration with the following command.
```

**Review by stoader — APPROVED** [review]

**Pull request merged**